researchstack-sdk/src/main/java/org/sagebionetworks/bridge/researchstack/BridgeDataProvider.java

```diff
@@ -4,8 +4,10 @@
 import android.support.annotation.NonNull;
 import android.support.annotation.Nullable;
 
+import com.fatboyindustrial.gsonjodatime.Converters;
```

Comment:
Delete. I don't see this used here, and I plan to remove this dependency from rest-client.